src/common/board.rs

```diff
@@ -56,14 +70,28 @@ pub trait Board: Status {
     fn get_digital_interrupt_value(&self, _pin: i32) -> anyhow::Result<u32> {
         anyhow::bail!("this board does not support digital interrupts")
     }
+    /// Get the pin's given duty cycle
     fn get_pwm_duty(&self, pin: i32) -> f64;
+    /// Set the pin to the given duty cycle 
```

Comment:
Should specify here (and in L. 73) that the domain for duty cycle is a percent value float between 0 and 1